summarizer/summarize.test.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const summarize = require('./summarize');
+
+describe('summarize tests...', () => {
+  const text = fs.readFileSync('./summarizer/text.txt', 'utf-8');
+  var data;
+
+  it('summarize returns string', async () => {
+    data = await summarize(text, 5);
+    expect(typeof data).toBe('string');
```

Comment:
Should we also test for the exact summary?

Reply:
I think it's a good idea, but I'm am not certain I have the best approach to it. I think I could compare how similar the machine summary is to a human summary and have it pass if it meets a threshold, but this requires a good human summary for each blog input.

Reply:
What I mean is: run your code once, get the summary it produces, see if you think it makes sense.  If it does, hard code that into your test, so we know in the future if it breaks for this given blog post text.  This is really a regression test, to guard against future problems.

Reply:
Alright, I added the test.